pandas/tests/test_algos.py

```diff
@@ -456,9 +457,10 @@ def test_datetime64tz_aware(self):
         result = Series(
             Index([Timestamp('20160101', tz='US/Eastern'),
                    Timestamp('20160101', tz='US/Eastern')])).unique()
-        expected = np.array([Timestamp('2016-01-01 00:00:00-0500',
-                                       tz='US/Eastern')], dtype=object)
-        tm.assert_numpy_array_equal(result, expected)
+        expected = DatetimeArray._from_sequence(np.array([
```

Comment:
I changed this without thinking about it... Do we want to change this to be a DatetimeArray or keep it as an ndarray of objects? I'm leaning towards ndarray of objects.